service/src/main/java/com/myhome/services/springdatajpa/MailSDJpaService.java

```diff
@@ -0,0 +1,46 @@
+package com.myhome.services.springdatajpa;
+
+import com.myhome.domain.User;
+import com.myhome.services.MailService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.MailSendException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class MailSDJpaService implements MailService {
```

Comment:
Would be great to create another implementation of MailService that will be printing the 'emails' into console to not use SMTP on dev server